src/calculateRentalCost.js

```diff
@@ -20,7 +20,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+  const dayilyPay = 40;
+
+  if (days < 3) {
+    total = days * dayilyPay;
+  } else if (days < 7) {
+    total = days * dayilyPay - 20;
```

Comment:
Добавляй скобки в выражения для улучшения читаемости
```suggestion
    total = (days * dayilyPay) - 20;
```